hplot/hstack.go

```diff
@@ -172,6 +197,11 @@ func (hs *HStack) hplot(c draw.Canvas, p *plot.Plot, h *H1D, yoffs []float64, hs
 		}
 	}
 
+	// Plot individual histo band when not stacked
+	if h.Band != nil && hsk == HStackOff {
```

Comment:
assuming we go with the `boolean/index` way from below, shouldn't we move this at the same place than for `HStackOff`?

Reply:
Actually, I think the one of `HStackOff` should move down (and the same for `hplot.H1D`). I think it's better to have the following order: fill, band and then line.